cornice_swagger/swagger.py

```diff
@@ -516,6 +517,7 @@ def from_schema_mapping(self, schema_mapping):
 
                 elif location in ('header', 'headers'):
                     header_schema = convert_schema(field_schema)
+                    print(header_schema)
```

Comment:
leftover

Reply:
Oops, that's from trying to debug Kinto/kinto#1503..